### PR TITLE
#60 Exception parsing invalid translation template

### PR DESCRIPTION
**Pull request by highsource**
- Added a reproducing test (parsing "Verkehrspädagoge")

This is a WIP PR, please do not merge yet.

### CODE REVIEW

**Comment by highsource** [comment]
Word done, this can be (please) merged.

File: src/test/resources/articles-de/Verkehrspaedagoge.txt [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+*{{es}}: [1] {{Ü|es|}}
+*{{cs}}: [1] {{Ü|cs|}}
+}}
+
+{{Referenzen}}
+:[1] {{Wikipedia|Verkehrspädagoge}}
+:[*] {{Ref-DWDS|Verkehrspädagoge}}
+
+{{Quellen}}
+</textarea><div class="templatesUsed"></div><p id="mw-returnto">Zurück zur Seite <a href="/wiki/Verkehrsp%C3%A4dagoge" title="Verkehrspädagoge">Verkehrspädagoge</a>.</p>
```
Comment (chmeyer) [inline src/test/resources/articles-de/Verkehrspaedagoge.txt]: Please remove html code that does not belong to the wiki markup (i.e., everything from `</textarea>`

File: src/test/java/de/tudarmstadt/ukp/jwktl/parser/de/DETranslationHandlerTest.java [hunk 2]
```diff
@@ -224,6 +224,11 @@ public void testBoot() throws Exception {
 		assertEquals("{{f}}", trans.get(2).getAdditionalInformation());
 	}
 
+	public void testParsesVerkehrspaedagoge() throws Exception {
+		IWiktionaryPage page = parse("Verkehrspaedagoge.txt");
+		assertEquals(1, page.getEntryCount());
```
Comment (chmeyer) [inline src/test/java/de/tudarmstadt/ukp/jwktl/parser/de/DETranslationHandlerTest.java]: I suggest to test at least for the number of translations parsed

File: src/test/resources/articles-de/Verkehrspaedagoge.txt (hunk 1 above)
Comment (highsource) [reply src/test/resources/articles-de/Verkehrspaedagoge.txt]: Right, sorry. My bad.

File: src/test/java/de/tudarmstadt/ukp/jwktl/parser/de/DETranslationHandlerTest.java (hunk 2 above)
Comment (highsource) [reply src/test/java/de/tudarmstadt/ukp/jwktl/parser/de/DETranslationHandlerTest.java]: No translations are parsed, but we can check for `isEmpty()`.

**Comment by highsource** [comment]
@chmeyer Thank you for the review, I've fixed both points.

**Pull request merged**